interfaces/portal/src/app/services/log.service.ts

```diff
@@ -76,15 +47,8 @@ export class LogService {
     this.appInsightsInitialized = true;
   }
 
-  public logPageView(name?: string): void {
-    if (this.appInsightsInitialized) {
-      this.appInsights.trackPageView({ name });
-    }
-    console.info(`LOG: PageView: "${name ?? ''}"`);
-  }
-
   public logEvent(
-    name: LogEvent,
+    name: string,
```

Comment:
what is the point of keeping this method (and the other unused methods here) if they are not used?